source/_includes/common-tasks/backups.md

```diff
@@ -1,6 +1,8 @@
 ## Backups
 
-Backup of your Home Assistant and add-on data and configuration. They are stored in a compressed archive file (.tar). Backups are made from the backups panel under {% my supervisor_backups title="**Settings** > **System** > **Backups**" %}. There is also a service available that allows you to trigger the creation of a backup from an automation. By default, backups are stored locally in the `/backup` directory.
+Backup of your Home Assistant and add-on data and configuration. Backups are used to [restore](#restoring-a-backup) a system or parts of it if a rollback is needed or to migrate your Home Assistant to new hardware. It is good practice to create a backup before updating Home Assistant.
```

Comment:
```suggestion
Backup of your Home Assistant and add-on data and configuration. Backups are used to [restore](#restoring-a-backup) a system or parts of it if a rollback is needed or to migrate your Home Assistant to new hardware. It is good practice to create a backup before updating.
```
There is no reason to limit this to Home Assistant.
When you update your add-ons as well, you should take a backup